src/main/java/uk/gov/hmcts/divorce/document/model/DocumentType.java

```diff
@@ -31,7 +31,7 @@
     BAILIFF_CERTIFICATE_OF_SERVICE("Bailiff certificate of service"),
 
     @JsonProperty("bailiffService")
-    @JsonAlias({"serviceBaliff"})
+    @JsonAlias({"serviceBailiff"})
```

Comment:
this need not change as `serviceBaliff` was removed and whichever case is having that will map to this option